Makefile

```diff
@@ -4,6 +4,9 @@ DJANGO_SETTINGS_MODULE = hypha.settings.dev
 JS_VENDOR_DIR = ./hypha/static_src/javascript/vendor
 JS_ESM_DIR = ./hypha/static_src/javascript/esm
 
+# check if pip install present, else try to use uv pip
+PIP := $(shell command -v pip 2> /dev/null || echo 'uv pip')
```

Comment:
Will not `command -v pip` be true as default on most systems? Can we check for `uv` instead and use `pip` only if it is not installed?